README.md

```diff
@@ -15,3 +15,185 @@ To run the tests:
 ```bash
 yarn test
 ```
+
+## Concepts
+
+`directo` is structured as parser for English imperative sentences. All commands must start with a `verb`. Recognized command structures are:
+
+- `verb`
+  - login
+  - exit
+- `verb object`
+  - go north
+  - buy sword
+  - get lamp
+- `verb preposition object`
+  - sit on the log
+- `verb object preposition subject`
+  - give sword to steve
+- `verb determiner object`
+  - eat the berries _Many of these are likely the same as `verb object`, but quantifiers make a difference here._
+- `verb subject determiner object`
+  - give steve the sword
+  - give steve a mushroom
+
+When using `directo`, commands are declared based on the set of verbs and command structure they accept. Many command implementations will want to handle multiple structures (see the two ways one could give a sword to steve above).
+
+## Usage
+
+Using `directo` is as simple as registering commands and handlers:
+
+```javascript
+const { command, processText, Format, CommandResult } = require('directo');
+
+command({
+  verb: 'go',
+  func: async ({ format, object }) => {
+    if (format !== Format.VO) {
```

Comment:
I'm assuming `Format.VO` refers to the aforementioned `verb object` command format. Are you just using `VO` to shorten things? I'm thinking it might be nice to use the full words in this case. Let's discuss.

Reply:
Yup to the shortening. Some of the longer ones would get ugly. `VERB_SUBJECT_DETERMINER_OBJECT` and whatnot. I'd be fine switching it to the longer form if we would prefer, though.

Reply:
That's a fair point. I suppose as long as we have it documented (which you do), the abbreviations are fine.